charts/opentelemetry-operator/templates/deployment.yaml

```diff
@@ -74,16 +74,16 @@ spec:
             {{- if .Values.manager.featureGates }}
             - --feature-gates={{ .Values.manager.featureGates }}
             {{- end }}
-            {{- if .Values.manager.createRbacPermissions }}
```

Comment:
Please add a case to `NOTES.txt` that fails if `manager.createRbacPermissions` is set

Reply:
@SergK I thought that this was the only place `createRbacPermissions` was used but I was wrong.  Since we still use the field in clusterrole.yaml your are correct that we dont need a failure case.

Your readme additions would be great.

Reply:
@TylerHelmuth I have updated the README.md file. Please let me know if any changes are needed in the section or anything else.